databases/mycli/Portfile

```diff
@@ -1,14 +1,12 @@
-# $Id$
```

Comment:
It's not absolutely necessary, but you could add the modeline on top. You'll also notice that [current version](https://github.com/xeron/macports-ports/blob/c2379d1ca9d637f4696c3f800c0e04b500f86ea8/databases/mycli/Portfile) doesn't show any syntax highlighting.